python_modules/dagster/dagster/_core/execution/backfill.py

```diff
@@ -49,6 +49,8 @@ class PartitionBackfill(
             ("backfill_timestamp", float),
             ("error", Optional[SerializableErrorInfo]),
             ("asset_selection", Optional[Sequence[AssetKey]]),
+            ("title", Optional[str]),
```

Comment:
we should do some validation on the title. There isn't a precedent for this that i'm aware of so:
1. what's a reasonable length limit for the title? 100 characters?
2. any limits on allowable characters? maybe restrict whitespace to only spaces?

Reply:
Ahh +1!  In the past we haven't set guard rails on things like this and it's come back to bite us -- it'd be great to make these 100 or 255 characters max.

It might be nice to exclude `"` so that a future UI could support using it for exact matching, and `*` and `%` for the same reasons 👀